README.md

```diff
@@ -1,3 +1,9 @@
+<p>
+    <a href="https://crates.io/crates/near-gas"><img src="https://img.shields.io/crates/dv/near-gas?style=flat-square&logo=near&label=crates.io" alt="Crates.io (latest)"></a>
```

Comment:
What does "21" (the number displayed on the badge) means?

Reply:
number of downloads

Reply:
```suggestion
    <a href="https://crates.io/crates/near-gas"><img src="https://img.shields.io/crates/d/near-gas?style=flat-square&logo=near&label=crates.io" alt="Crates.io (downloads)"></a>
```